mixer/adapter/stackdriver/contextgraph/send.go

```diff
@@ -18,11 +18,14 @@ import (
 	"context"
 	"time"
 
+	"github.com/gogo/protobuf/proto"
+	"github.com/gogo/status"
 	"github.com/golang/protobuf/ptypes/timestamp"
 	contextgraphpb "google.golang.org/genproto/googleapis/cloud/contextgraph/v1alpha1"
-	"google.golang.org/grpc/status"
 )
 
+const maxReq = 550 * 1024
```

Comment:
nit: maybe add some documentation on what this number means and where the limits come from?

Reply:
Done.